src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav-links {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-links .nav-link-item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
Be consistent with your vertical margins. It's a good practice to add only top or only bottom margin, but not both. In this case, you've added a right margin to the `.nav-link-item` class. Instead, consider adding a left margin to all `.nav-link-item` classes except the first one. This approach will make your code more consistent.